Vagrantfile

```diff
@@ -100,6 +103,9 @@ Vagrant.configure("2") do |global_config|
       if vagrant_boxes.key? vagrant_box
         config.vm.box_url = vagrant_boxes[vagrant_box]
       end
+      if vagrant_box_aliases.key? vagrant_box
+        config.vm.box = vagrant_box_aliases[vagrant_box]
```

Comment:
I think `box_url` might be deprecated for app.vagrantup hosted boxes - we probably need to migrate "most" of the boxes to these "aliases"